docs/migration-guide.md

```diff
@@ -137,94 +366,41 @@ To avoid conflicts when merging your app resources, all resources declared in th
 | `r2_media_notification_channel_description` | `readium_media_notification_channel_description` |
 | `r2_media_notification_channel_name`        | `readium_media_notification_channel_name`        |
 
-#### Drawables
+##### Renamed drawables
 
 | Deprecated                              | New                                          |
 |-----------------------------------------|----------------------------------------------|
 | `r2_media_notification_fastforward.xml` | `readium_media_notification_fastforward.xml` |
 | `r2_media_notification_rewind.xml`      | `readium_media_notification_rewind.xml`      |
 
-### Publication assets
-
-In most cases, you no longer need to manually create a `PublicationAsset` to open a publication with
-the streamer. You can use the overloaded open method taking a `Url` as argument instead.
 
-```kotlin
-streamer.open(file.toUrl(), ...)
-```
-
-### Tap and drag events
+## 2.4.0
 
-The `VisualNavigator.Listener`'s `onTap` and `onDrag` events are deprecated. You can now provide multiple implementations of `InputListener`, and the order matters if events are consumed.
+### Maven Central
 
-```kotlin
-navigator.addInputListener(DirectionalNavigationAdapter())
+Readium is now distributed with [Maven Central](https://search.maven.org/search?q=g:org.readium.kotlin-toolkit). You must update your Gradle configuration.
 
-navigator.addInputListener(object : InputListener {
-    override fun onTap(navigator: VisualNavigator, event: TapEvent): Boolean {
-        toggleUi()
-        return true
+```diff
+allprojects {
+    repositories {
+-       maven { url 'https://jitpack.io' }
++       mavenCentral()
     }
-})
+}
 ```
 
-### New navigator interfaces
-
-An important part of `VisualNavigator` was moved to `DirectionalNavigator`. A new `HyperlinkNavigator`
-interface implemented by the `EpubFragmentNavigator` provides the ability to intercept clicks on
-links to both internal and external URLs. Opening external URLs is no longer handled by the navigator,
-that's something you should do by yourself in the way you like.
```

Comment:
Is this still elsewhere?

Reply:
There are `@Deprecated` notices that will explain the moves. For explaining the new interfaces, I think we can do that in the Navigator user guide.